auto_backup/tests/test_db_backup.py

```diff
@@ -175,18 +175,6 @@ def test_action_backup_sftp_remote_open(self):
                     'wb'
                 )
 
-    def test_action_backup_sftp_remote_open(self):
```

Comment:
Why this test was removed?

Reply:
Hello, 
I found that it is the same test than the previous test

Reply:
Your're right